lib/saxerator/builder/xml_builder.rb

```diff
@@ -1,5 +1,11 @@
+require 'rexml/document'
+
 module Saxerator
   module Builder
+    class XmlDocument < REXML::Document
```

Comment:
I'm for removing it and simply returning the REXML::Document object. Technically this library isn't 1.0, so this is a tolerable breaking change.